api_docgen/odoc/Makefile

```diff
@@ -190,3 +194,24 @@ install:
 	if test -d build/man/compilerlibref ; then \
 	  $(INSTALL_DATA) build/man/libref/* "$(INSTALL_LIBRARIES_MAN_DIR)"; \
 	fi
+
+# Dependencies for stdlib modules.
+# Use the same dependencies used for compiling .cmx files.
+# The existing rules look like this:
+#   stdlib__X.cmx: x.ml \
+#       stdlib__Y.cmx \
+#       stdlib__X.cmi
+# We want:
+#   build/libref/stdlib__X.odoc: \
+#       build/libref/stdlib__Y.odoc \
+#       stdlib__X.cmti
+build/.depend: ../../stdlib/.depend | build/
+	sed \
```

Comment:
I noticed `sed` was already used during the build but perhaps an ocaml script would be more appropriate wrt maintenance ?

Reply:
I think we're trying to converge on python for this kind of script.

Reply:
There's currently no python script in the sources, did you mean ocaml ?

Reply:
We are in fact planning to *not* use python, because its installation is problematic on WIndows. Using `sed` for now is fine. 

I am wondering it we should update ocamldep to emit the right dependency information directly at some point.